src/core/src/bootstrap/Constants.py

```diff
@@ -98,6 +98,27 @@ class ConfigSettings(EnumBackport):
     # patch versions for healthstore when there is no maintenance run id
     PATCH_VERSION_UNKNOWN = "UNKNOWN"
 
+    # Strings used in perf logs
+    class LogStrings:
+        TASK = "task"
+        TASK_STATUS = "task_status"
+        PACKAGE_MANAGER = "package_manager"
+        NUMBER_OF_TRIALS = "number_of_trials"
+        NUMBER_OF_ROUNDS = "number_of_rounds"
+        FAILED = "failed"
+        SUCCEEDED = "succeeded"
```

Comment:
Since Failed and Succeeded are used to set the value for STATUS or (TASK_STATUS in the PR currently), better to have an enum for these two values.

Reply:
Done